CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/ArrangementPluginRegistry.java

```diff
@@ -34,6 +34,7 @@
 import au.gov.asd.tac.constellation.plugins.arrangements.random.RandomArrangementPlugin;
 import au.gov.asd.tac.constellation.plugins.arrangements.resize.ContractGraphPlugin;
 import au.gov.asd.tac.constellation.plugins.arrangements.resize.ExpandGraphPlugin;
+import au.gov.asd.tac.constellation.plugins.arrangements.scatter3d.ArrangeInScatter3dGeneralPlugin;
```

Comment:
For consistency with other arrangement plugin, could you rename the package to just `scatter` - there is no reason we couldn't have a Scatter2D plugin alongside this one.

Reply:
Done.